# sensor_height: 0.83
ground_height_tolerance: 0.3
ground_normal_deviation: 0.2
intensity_gradient: 200.0
texture_intensity: 5.0
intensity_tolerance: 5.0
terrain_number: 2
x_range: 10
y_range: 10


pb_ground_topic: ground_points
pb_road_topic: road_points

input_pointcloud: /velodyne_points